tools/ci_build/github/linux/docker/Dockerfile.manylinux2010_cpu

```diff
@@ -1,11 +1,11 @@
-FROM quay.io/pypa/manylinux2010_x86_64:2020-04-04-f427f46
```

Comment:
we pinned it to a specific version because we encountered some bug which broke the pipelines.
i assume you want to pull a more recent image, but wouldn't it be safer to pin to the exact version which you've validated?

Reply:
As we push the image to our private repo, it doesn't matter anymore. It won't suddenly break our build.